classes/class-ade-edd-loader.php

```diff
@@ -53,7 +53,7 @@ public function __construct() {
 		 */
 		public function ade_script() {
 
-			wp_register_script( 'ade-edd-js', ADE_EDD_PLUGIN_URL . '/assets/js/ade_load_js.js', array( 'jquery' ), '1.0.1', true );
+			wp_register_script( 'ade-edd-js', ADE_EDD_PLUGIN_URL . '/assets/js/ade_load_js.js', array( 'jquery' ), '1.0.2', true );
```

Comment:
Can you simply add a constant here instead of incrementing version number manually everytime?

Reply:
Added the constant.